### PR TITLE
[CDAP-6210] Adds new footer image

### PR DESCRIPTION
**Pull request by PGuimarais**
new logo footer

https://issues.cask.co/browse/CDAP-6210

Bamboo Build:
http://builds.cask.co/browse/CDAP-DRC4195/

### CODE REVIEW

File: cdap-ui/app/styles/themes/cdap.less [hunk 1]
```diff
@@ -335,11 +335,14 @@ body.theme-cdap {
...
     span.cask-logo {
-      background: url('@{img-path}/cdap_sprite.png') -360px 0 no-repeat;
+      background: url('@{img-path}/CASK_footer.png') 0 0 no-repeat;
       display: inline-block;
-      height: 30px;
-      width: 26px;
+      height: 40px;
+      width: 45px;
       vertical-align: top;
+      position: relative;
```
Comment (elfenheart) [inline cdap-ui/app/styles/themes/cdap.less]: instead of setting position relative, just do negative margin-top

**Comment by ajainarayanan** [comment]
@PGuimarais The image looks pixelated. Didn't we get a svg or some other high-res image from Sean?

**Comment by elfenheart** [comment]
if you are not using the png file anymore, please delete

**Comment by elfenheart** [comment]
👍 LGTM

**Pull request merged**